helm/hpcc/templates/dafilesrv.yaml

```diff
@@ -89,5 +89,12 @@ kind: ConfigMap
 {{ include "hpcc.addService" ( dict "root" $ "name" .name "service" $service "selector" .name "defaultPort" 7600 ) }}
 ---
 {{ include "hpcc.addCertificate" (dict "root" $ "name" .name "service" .service "component" "dafilesrv" "visibility" .service.visibility) }}
+
+{{- $instance := .name -}}
+{{- $visibility := .service.visibility -}}
+{{- range $remoteClient := .remoteClients }}
+ {{ include "hpcc.addClientCertificate" (dict "root" $ "client" $remoteClient.name "instance" $instance "component" "dafilesrv" "visibility" $visibility) }}
+{{- end }}
```

Comment:
the cert. for foreign access may not be appropriate, given that it isn't possible to mix secure/non-secure client access from a foreign environments, and in near future remote won't use certs, but the existing MTLS and signature verificaion route.